web/src/styles/entrypoints/home.scss

```diff
@@ -280,7 +277,63 @@ a {
 @font-face {
   font-family: 'Montserrat', 'sans-serif';
 }
+@media only screen and (max-width: 768px) {
+  #Our_Player,#Developer,#Patron {
+    width: 90%;
+  }
+  #br,#go {
+    background-image:none;
+  }
+
+  .menu>li:first-child {
+    border-left: 0px;
+  }
 
+  .menu>li {
+    display: inline-block;
+    padding-left: 35%;
+    padding-right: 35%;
+    border-right: 0px;
+  }
+  #top-box {
+    background-size: 150%;
+    background-image: url('/static/image/gg.png');
+  }
+
+  .menu a:hover {
+    border-style:solid ;
+    border-width: 2px;
+    padding: 5px;
+  }
+}
+
+@media only screen and (max-width: 720px) {
+  #Our_Player,#Developer,#Patron {
+    width: 56%;
+  }
+  #top-box {
+    background-size: 210%;
+  }
+}
+@media only screen and (max-width: 480px) {
+  #Our_Player,#Developer,#Patron {
```

Comment:
The "Our_Player" id should be "our-player", "Developer" to "developer", "Patron" to "patron"